src/Data/Container.agda

```diff
@@ -40,11 +35,3 @@ module Morphism where
 
   open import Data.Container.Morphism
     using (id; _∘_) public
-
--- The least and greatest fixpoints of a container.
-
-μ : ∀ {s p} → Container s p → Set (s ⊔ p)
```

Comment:
This seems "too far" for me: `Data.W` is safe. It's only `Codata.M` which requires `--sized-types`. So I'd be tempted to be finer in the splitting, i.e. keep `μ` here.  And then only split off GreatestFixpoint.

Reply:
I was trying to avoid to much fragmentation, but you're right it's better to fragment than be precise here. I've pushed changes that do as you suggest :+1: